### PR TITLE
 WIP to add event telemetry

### PR DESCRIPTION
**Pull request by garvankeeley**

### CODE REVIEW

File: Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift [hunk 1]
```diff
@@ -70,6 +71,8 @@ class ContentBlockerSettingViewController: ContentBlockerSettingsTableView {
                 ContentBlockerHelper.prefsChanged()
 
                 LeanplumIntegration.sharedInstance.track(eventName: .trackingProtectionSettings, withParameters: ["Enabled option": option.rawValue as AnyObject])
+                Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.appSettingChange, object: TelemetryEventObject.trackingProtectionEnabled, value: option.rawValue)
```
Comment (justindarc) [inline Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift]: Can probably shorthand these enum values (i.e. `category: .action, method: .appSettingChange`, etc.)

Comment (justindarc) [reply Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift]: Oh wait, they're not enums, are they?

Comment (justindarc) [reply Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift]: Nevermind... back to bed 🤒

**Comment by garvankeeley** [comment]
Closing as this was mostly to demonstrate this API

**Pull request closed without merging**